lightning/src/ln/channelmonitor.rs

```diff
@@ -735,6 +735,8 @@ pub struct ChannelMonitor<ChanSigner: ChannelKeys> {
 	commitment_transaction_number_obscure_factor: u64,
 
 	destination_script: Script,
+	revokeable_script: Option<(Script, SecretKey, Script)>,
```

Comment:
Maybe `broadcasted_local_revokable_script` to indicate both when it is filled in and who's it is?